CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.13.0
+
+### Technical improvements
+
+- Integrated new rendering engine for Detection, Segmentation, Lane visualizations & AR
```

Comment:
Well, I guess there should be `Lane Detection` then.
However I'm already a bit confused so feel free to leave as is.

Reply:
@kiryldz How about `Integrated new rendering engine for Object & Lane Detection, Segmentation, AR`?

Reply:
I like it!